### PR TITLE
Add ability to read forsys input parameters from the db given a project_id

### PR DESCRIPTION
**Pull request by elsiel23**

### CODE REVIEW

File: src/planscape/forsys/get_forsys_inputs_test.py [hunk 1]
```diff
@@ -154,10 +162,55 @@ def test_raises_error_for_url_params_project_areas_missing_id_field(self) -> Non
...
+        self.condition2 = Condition.objects.create(
+            condition_dataset=self.base_condition, raster_name="name2")
+
+        self.user = User.objects.create(username='testuser')
+        self.user.set_password('12345')
+        self.user.save()
+
+        self.geometry = {'type': 'MultiPolygon',
+                         'coordinates': [[[[1, 2], [2, 3], [3, 4], [1, 2]]]]}
+        stored_geometry = GEOSGeometry(json.dumps(self.geometry))
```
Comment (strangeties) [inline src/planscape/forsys/get_forsys_inputs_test.py:181]: to fix the failing test: replace with "self.stored_geometry ="

File: src/planscape/forsys/get_forsys_inputs_test.py [hunk 2]
```diff
@@ -154,10 +162,55 @@ def test_raises_error_for_url_params_project_areas_missing_id_field(self) -> Non
...
+        self.plan_with_user = Plan.objects.create(
+            owner=self.user, name="plan", region_name='sierra_cascade_inyo', geometry=stored_geometry)
+
+        self.project_with_user = Project.objects.create(
+            owner=self.user, plan=self.plan_with_user, max_cost=100, )
+        self.project_with_user.priorities.add(self.condition1)
+        self.project_with_user.priorities.add(self.condition2)
+
+        self.project_area_with_user = ProjectArea.objects.create(
+            owner=self.user, project=self.project_with_user, project_area=stored_geometry, estimated_area_treated=200)
```
Comment (strangeties) [inline src/planscape/forsys/get_forsys_inputs_test.py:191]: nit: please run formatter on this file

**Review by strangeties — APPROVED** [review]
thanks for making this change!

File: src/planscape/forsys/get_forsys_inputs_test.py (hunk 1 above)
Comment (elsiel23) [reply src/planscape/forsys/get_forsys_inputs_test.py:181]: thanks!

**Pull request merged**